backend/application/syson-sysml-import/pom.xml

```diff
@@ -84,6 +84,11 @@
 			<version>2025.4.5</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
Please update the version.
Please move it before tests dependencies.

Reply:
done